.github/workflows/hitl.yml

```
@@ -16,3 +16,12 @@ jobs:
           event-type: stabilizer
           repository: quartiq/hitl
           client-payload: '{"github": ${{ toJson(github) }}}'
+
+      - name: Wait for startup
+        run: sleep 30
+
+      - uses: fountainhead/action-wait-for-check@v1.0.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checkName: HITL
```

```suggestion
          checkName: HITL Run Status
```

* Can we move the `cargo flash` step into `hitl/run.sh`?
* Where is the `details_url` ending up? It would be nice to have that link from the commit checks on `stabilizer` to the hitl run on `hitl` (for those who can access it).

---

Re: `details_url` - this appears to be a broken portion of the GHA API: https://github.com/LouisBrunner/checks-action/issues/18